container/Exchange.js

```diff
@@ -18,7 +34,7 @@ export default () => {
   useEffect(() => {
     let cancelled = false
 
-    setLoading(true)
+    setLoading(<Spinner />)
```

Comment:
I think this could be a boolean again as `loading` is only used as a boolean

Reply:
Sorry my bad, I forgot to change it back. Thanks for the heads up!